tests/integration/cassandra_tests.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+import time
+import unittest
+
+try:
+    from cassandra import InvalidRequest
+    from cassandra.cluster import Cluster
+    from cassandra.query import dict_factory, named_tuple_factory
+except ImportError:
+    raise unittest.SkipTest("cassandra-driver is not installed")
```

Comment:
Yeah, I'm not sure what the right tack to take here is. This protects us in weird build environments like launchpad but at the cost of potentially hiding errors. The skips are at least loudly announced in the output of test run.

The dev environment (as set up by vagrant+puppet) will have a C* and all the relevant libs so things shouldn't get missed if people are running tests locally. I am a bit concerned that I haven't got C* running in Travis though.